external-tests/performance/test.sh

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+
+export TESTS_REPO=https://github.com/openshift-kni/performance-addon-operators
+export TESTS_LOCATION=/tmp/performance-operators
```

Comment:
We have a _cache directory within the source tree we could use. Or are we trying to avoid usage of that directory because of having nested repos?

Reply:
Yes, I wanted to be outside this repo path. Does it make sense / do we have alternatives?

Reply:
/tmp is fine i suppose. i don't have a better solution.